.travis.yml

```diff
@@ -45,7 +45,10 @@ matrix:
         - os: osx
           env: SETUP_CMD='test'
                CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES
-               PIP_DEPENDENCIES='jplephem'
+               if [[ $PYTHON_VERSION == 2.7 ]]; 
+                 then PIP_DEPENDENCIES='jplephem mock';
```

Comment:
this looks a very weird formatting, could you the usual one, putting else on the same level of if?